libr/util/file.c

```diff
@@ -1049,7 +1052,16 @@ R_API bool r_file_copy (const char *src, const char *dst) {
 #if HAVE_COPYFILE_H
 	return copyfile (src, dst, 0, COPYFILE_DATA | COPYFILE_XATTR) != -1;
 #elif __WINDOWS__
-	return r_sys_cmdf ("copy %s %s", src, dst);
+	char *esrc = r_str_replace (strdup (src), "\"", "^\"", 1);
+	char *edst = r_str_replace (strdup (dst), "\"", "^\"", 1);
+	char *s = r_file_abspath (esrc);
+	char *d = r_file_abspath (edst);
+	bool ret = r_sys_cmdf ("copy \"%s\" \"%s\"", s, d);
```

Comment:
Is there a better way to do this?

Reply:
do this works? because last time i tried quoting things in  windows wasnt workng at all, will be better to have an r_sys_cmdv() with variable argument handling to get argv as separate parameters instead of passing just a string and then using spawn instead of system to explictly separate each argument

Reply:
but this function doesnt exist, so the fix requires more work

Reply:
only double quotes work in windows cmd, I tested and the only problem now is paths with spaces. But thats a problem in linux as well

Reply:
I fixed the path with spaces problem using r_str_argv(). can you please check?

Reply:
thats not how to use r_str_argv, imho the whole thing is getting overengineered and the solution will be simpler if done like i said at the begining